test/upgrade/prober/configuration.go

```diff
@@ -39,6 +43,7 @@ const (
 	configFilename   = "config.toml"
 	watholaEventNs   = "com.github.cardil.wathola"
 	healthEndpoint   = "/healthz"
+	brokerClass      = "MTChannelBasedBroker"
```

Comment:
Do you think it makes sense to make this configurable through env var too? This way the file can be more flexible in terms of which broker class to test.

Reply:
We can change this to `defaultBrokerCalss`, then check an env var like `UPGRADE_TEST_BROKER_CLASS` in `deployBroker`.